README.md

```diff
@@ -1,65 +1,61 @@
+To run resource estimation using Azure Quantum Resource Estimation (QRE) tool, one needs to have Azure QRE package configure. See [this tutorial](https://learn.microsoft.com/en-us/azure/quantum/intro-to-resource-estimation) for more information.
 
 ## Usage
-
-Please take a look at the `examples` directory.
-We have multiple examples there:
-- `h_chain_trotter.py` shows how to use graph state compilation on a simple hydrogen chain example. (Requires `pyscf` install extra.)
-- `resource_estimate_from_qasm.py` shows how to use graph state compilation when the circuit is loaded from QASM.
-- `qsp_vlasov.py` shows how to perform resource estimation.
+See the [`examples`](examples) directory to learn more about how to use Bench-Q.
 
 ## Running benchmarks
 
-To run the benchmarks run
+Because quantum compilation and resource estimation can be compute intensive, Bench-Q includes tools for benchmarking components that are potential bottlenecks. To run the benchmarks, execute the command
 
 ``` bash
 pytest benchmarks/
 ```
 
-from the top-level directory of this repo. By default, this will skip some benchmarks that run extremely low. If you want to run
+from the top-level directory of this repo.
+
+By default, this will skip some benchmarks that are extremely slow. If you want to run
 those too, set environmental variable `SLOW_BENCHMARKS` to any value, e.g.:
 
 ``` bash
 SLOW_BENCHMARKS=1 pytest benchmarks/
 ```
 
-Our benchmarks are run automatically on each release. You can see the performance of benchq over time on [benchq's benchmark page](https://zapatacomputing.github.io/benchq/dev/bench/).
+These benchmarks are run automatically on each release. You can see the performance of benchq over time on [benchq's benchmark page](https://zapatacomputing.github.io/benchq/dev/bench/).
 
 ## Development and Contribution
 
-To install the development version, run `pip install -e '.[dev]'` from the main directory. (if using MacOS, you will need single quotes around the []. If using windows, or Linux, you might not need the quotes).
+To install the development version, run `pip install -e '.[dev]'` from the main directory.
 
-We use [Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) docstring format. If you'd like to specify types please use [PEP 484](https://www.python.org/dev/peps/pep-0484/) type hints instead adding them to docstrings.
+We use [Google-style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) docstring format. If you'd like to specify types, please use [PEP 484](https://www.python.org/dev/peps/pep-0484/) type hints instead adding them to docstrings.
 
 There are codestyle-related [Github Actions](.github/workflows/style.yml) running for PRs. (TODO)
```

Comment:
What is the TODO at the end of this line? Should we get rid of it?

Reply:
I'm not sure, so I've just removed it.